private-templates-service/client/src/components/AdaptiveCard/AdaptiveCard.tsx

```diff
@@ -0,0 +1,71 @@
+import * as React from "react";
+import * as AdaptiveCards from "adaptivecards";
+import { Card, Container, TemplateName } from "../AdaptiveCard/styled";
+
+function getCard() {
+	// Hard coded, will remove and connect to backend in future PR
```

Comment:
FYI for the future, when you're removing the hard-coded data, you'll want to create an interface, which defines the structure of a card. For example:

```
interface CardData {
	type: boolean;
	version: string;
	body: any;
	actions: any;
}
```

Reply:
Do you have any suggestions on how to make an interface for the adaptive card if I don't know what's going to be in the card?

Reply:
Ohh, I thought this was the standard structure across all Cards! If the object doesn't have consistent keys, then it won't make sense to base it on an interface. So it's good as it is :)